src/main/java/org/jitsi/utils/logging2/Logger.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.utils.logging2;
+
+import java.util.*;
+import java.util.logging.*;
+
+public interface Logger
+{
+    /**
+     * Create a 'child' logger which derives from this one.  The child logger
+     * will share the same log level setting as this one and its
+     * {@link LogContext} (given here) will inherit all the context
+     * from this logger.
+     * @return
+     */
+    Logger createChildLogger(String name, Map<String, String> context);
```

Comment:
What does the two-argument method do?

Reply:
I meant it needs a doc because it's not obvious (since this doc says it inherits the context of the parent, but there's also a context passed in)